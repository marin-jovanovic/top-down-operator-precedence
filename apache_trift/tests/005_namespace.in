namespace c_glib _test
