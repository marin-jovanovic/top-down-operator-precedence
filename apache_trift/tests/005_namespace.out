Document
 HeaderManager
  Header
   Namespace
    "namespace"
    NamespaceScope
     c_glib
    Identifier
     _test
  HeaderManager
   end of header instructions
 DefinitionManager
  end of definition instructions
